.github/workflows/nodejs.yml

```diff
@@ -20,6 +20,8 @@ jobs:
     - name: npm install, build, and test
       run: |
         npm ci
-        npm test
+        time npm test
+        npm run lint || true
```

Comment:
can be enabled (fail when lint fail) once the test directory is fixed (there are some lint errors)

Reply:
Isn't there a separate GitHub app that lints? Would be nice to have a separate check for this..